### ISSUE TITLE
Performance report incorrectly mentions "excessive logging"

### ISSUE THREAD

**Issue by szhorvat** [google/oss-fuzz#7314]
In the libFuzzer performance reports I often see complaints about excessive logging, saying: "The fuzz target writes too many log messages (either stdout or stderr)." However, the log files that the reports link to show no output by the program. In fact I am quite certain that there is no output, other than libFuzzer's own.  Is this a known/common problem?

Here are two such example reports:

https://oss-fuzz.com/performance-report/libFuzzer_igraph_bliss_fuzzer/libfuzzer_asan_igraph/latest
https://oss-fuzz.com/performance-report/libFuzzer_igraph_vertex_separators_fuzzer/libfuzzer_asan_igraph/latest

**Comment by jonathanmetzman** [comment]
I think youre right about this, I think this feature is buggy.
I can try to take a look at this, but I'm not sure it's a very high priority for us.

**Comment by jonathanmetzman** [comment]
Yeah basically we track the amount of lines printed to stderr/stdout and report them.
In AFL, the fuzzing engine code prints nothing, so it is reported correctly as 0.
In libFuzzer, libFuzzer writes to stderr so the code has to correctly figure out what is from libFuzzer and what isn't.
This code is not doing its job correctly.
Anyway I think this stat is fairly silly to begin with, devs can figure out on their own if the fuzz target is logging, we don't really need to report it to them. And although logging hurts performance, IMO it doesn't hurt it so much that it deserves this prominence.
